client/landing/jetpack-cloud/sections/scan/main.jsx

```diff
@@ -3,22 +3,103 @@
  */
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import { Button } from '@automattic/components';
+import { translate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { withLocalizedMoment } from 'components/localized-moment';
+import SecurityIcon from 'landing/jetpack-cloud/components/security-icon';
+
+import './style.scss';
 
 class ScanPage extends Component {
+	renderScanOkay() {
+		const { siteSlug, moment, lastScanTimestamp, nextScanTimestamp } = this.props;
+
+		let nextScanString = translate( 'tomorrow' );
+		if ( moment( nextScanTimestamp ).isSame( Date.now(), 'd' ) ) {
+			nextScanString = translate( 'later today' );
+		}
+
+		return (
+			<>
+				<SecurityIcon className="scan__icon" />
+				<h1 className="scan__header scan__header--okay">
+					{ translate( 'Don’t worry about a thing' ) }
+				</h1>
+				<p>
+					{ translate(
+						'The last Jetpack scan ran {{strong}}%(time)s{{/strong}} and everything looked great.' +
+							' Run a manual scan now or wait for Jetpack to scan your site %(nextScan)s.',
+						{
+							args: {
+								time: moment( lastScanTimestamp ).fromNow(),
+								nextScan: nextScanString,
+							},
+							components: { strong: <strong /> },
+						}
+					) }
+				</p>
+				<Button
```

Comment:
This might need to be a feature flagged. Since I am not 100% sure if we will be able to run Manual Scans.

Reply:
Done with 88109832b01f858f5ee2e8e09b1c21bd12fc5cc2.